File: 프로그래머스/lv1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.py
def solution(price, money, count):
    answer = -1
    
    # 이용금액 , 가지고 있는돈, 놀이공원 탄 횟수
    # n 배 씩 증가
    
    total = 0
    for i in range(1,count+1):
        total += price * i
    
    answer = total - money
    
    if answer < 0:
        answer = 0
    return answer

File: 프로그래머스/lv1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js
function solution(price, money, count) {
    var answer = -1;
    
    let total = 0
    for (let i = 1 ; i < count+1 ; i ++) {
        total += price * i
    }
    
    answer = total - money
    if (answer < 0) { answer = 0 }
    return answer;
}